src/objects/Object.h

```diff
@@ -0,0 +1,30 @@
+#ifndef OBJECT_H
+#define OBJECT_H
+
+#include "stdlib.h"
+#include <iostream>
+
+class Level;
+
+class Object {
+private:
+    Level* level;
```

Comment:
Если уровень не может исчезнуть за время жизни объекта, то используй ссылку вместо указателя, я и так там пытаюсь довести до ума пул реквесты с заменой сырых указателей на ссылки или умные указатели, где требуется.
```suggestion
    Level& level;
```